SparkPost/2web2ui
# Seedlist scaffold

Opening this for collaboration to avoid too many merge conflict

How to Test:
- Start seedlist locally and point to it on openresty
- In current codebase, the endpoint is `/api/v1/inbox-placement/seedlist`; rename it to `/api/v1/inbox-placement/seeds` (according to api doc). 
- Unless you've working project, you can modify `controller.getSeedlist` (in `resources/controller.js`) to return a  list of addresses

Notes: 
- Depends on https://github.com/SparkPost/accusers-api/pull/675/files to use the account option. Account option visibility is set wrong
- Need to confirm path https://github.com/SparkPost/sparkpost-admin-api-documentation/pull/356#discussion_r308461529
